REUSE.toml

```diff
@@ -0,0 +1,22 @@
+version = 1
+SPDX-PackageName = "Gardener gardener-extension-shoot-oidc-service"
+SPDX-PackageSupplier = "The Gardener project <gardener@googlegroups.com>"
+SPDX-PackageDownloadLocation = "https://github.com/gardener/gardener-extension-shoot-oidc-service"
+
+[[annotations]]
+path = "**"
+precedence = "aggregate"
+SPDX-FileCopyrightText = "2017-2024 SAP SE or an SAP affiliate company and Gardener contributors"
```

Comment:
Now that we are at it should we keep maintaining the year?

Reply:
Don't know, just ran `reuse convert-dep5`, @rfranzke any suggestions?

Reply:
Feel free to change it to 2025 - the command just translates what was there before